lib/rails_admin/config/fields/types/citext.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_admin/config/fields/types/string'
+
+module RailsAdmin
+  module Config
+    module Fields
+      module Types
+        class Citext < String
```

Comment:
Wouldn't it better to use `Text` as the superclass?
https://github.com/railsadminteam/rails_admin/blob/master/lib/rails_admin/config/fields/types/text.rb

Reply:
That works for me. Done in latest.